common/reportMetrics.py

```diff
@@ -526,6 +526,9 @@ def replay_sar(metric_file_path, command):
         # parse opening lines
         line = metric_file.readline()
         header = line.split()
+        if len(header) < 4:
+            logger.error('Error in sar file header')
+            sys.exit(1)
         host = header[2][1:-1]
```

Comment:
header[2][1:-1] is to trim the first and last character.
ex. 3453465423 -> 45346542.
is this what you want?

Reply:
yes, the string is "(host)"